cloud/main.js

```diff
@@ -29,7 +29,7 @@ Parse.Cloud.define("hello", function(request, response) {
 //   "endTimeToken": "00000"}
 
 Parse.Cloud.define("replayEvents", function(request, response) {
-    var requiredParams = ["count", "aliasId", "subkey"];
```

Comment:
if you're going to remove count from the requiredParams, you'll have to update the dec below to

`var count = params.count ? params.count : DEFAULT_COUNT`

where `DEFAULT_COUNT` is 20 or something